hw2/04-background.html

```diff
@@ -6,23 +6,26 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
   <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
+    <main class="container vw-50 mt-4 bg-white rounded">
+      <h1 class="row p-3">Exercise 04 - Background</h1>
+      <p class="row mx-1">
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+      <div class="row d-flex align-items-center">
+        <label class="row mx-1">Background change interval (seconds): </label>
```

Comment:
you need a 'for' attribute on the label to connect it to the input to make this accessible